committee/committee.go

```diff
@@ -165,44 +165,41 @@ func NewCommittee(kvstore db.KVStore, cfg Config) (Committee, error) {
 		nextHeight:            cfg.GravityChainStartHeight,
 	}, nil
 }
-
-func (ec *committee) Start(ctx context.Context) (err error) {
+func (ec *committee) load(ctx context.Context) (err error) {
 	ec.mutex.Lock()
```

Comment:
we are not supposed to call lock in private function, as ppl may misuse it which will cause dead lock.